env/build.go

```diff
@@ -37,6 +56,26 @@ func matches(k1, k2 string) bool {
 	return k1 == k2
 }
 
+func isNotIncludedForAPI(platformAPI *api.Version) func(k string) bool {
+	if platformAPI.Compare(api.MustParse("0.8")) < 0 {
+		return isNotIncluded
+	}
```

Comment:
This could go away if we do https://github.com/buildpacks/lifecycle/pull/591/files#r625322067